test/jellyfish/component/hls/manager_test.exs

```diff
@@ -0,0 +1,87 @@
+defmodule Jellyfish.Component.HLS.ManagerTest do
+  @moduledoc false
+
+  use ExUnit.Case
+
+  import Mox
+
+  alias Jellyfish.Component.HLS
+  alias Jellyfish.Component.HLS.Manager
+
+  @files ["manifest.m3u8", "header.mp4", "segment_1.m3u8", "segment_2.m3u8"]
+  @body <<1, 2, 3, 4>>
+  @s3_credentials %{
+    access_key_id: "access_key_id",
+    secret_access_key: "secret_access_key",
+    region: "region",
+    bucket: "bucket"
+  }
+
+  setup do
+    room_id = UUID.uuid4()
+    hls_dir = HLS.output_dir(room_id, persistent: false)
+    options = %{s3: nil, persistent: true}
+
+    File.mkdir_p!(hls_dir)
+    for filename <- @files, do: :ok = hls_dir |> Path.join(filename) |> File.write(@body)
+
+    on_exit(fn -> File.rm_rf!(hls_dir) end)
+
+    {:ok, %{room_id: room_id, hls_dir: hls_dir, options: options}}
+  end
```

Comment:
Why do we generate a each time new UUID and generate a file instead of generating it once in `setup_all` and constant UUID?